lib/pages/settings/view/request_card.dart

```diff
@@ -0,0 +1,207 @@
+import 'package:acs_upb_mobile/authentication/model/user.dart';
+import 'package:acs_upb_mobile/generated/l10n.dart';
+import 'package:acs_upb_mobile/pages/settings/model/request.dart';
+import 'package:acs_upb_mobile/pages/settings/service/admin_provider.dart';
+import 'package:acs_upb_mobile/widgets/button.dart';
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:provider/provider.dart';
+
+class RequestCard extends StatefulWidget {
+  const RequestCard({this.requestId});
+
+  @required
+  final String requestId;
+
+  @override
+  _RequestCardState createState() => _RequestCardState();
+}
+
+class _RequestCardState extends State<RequestCard>
+    with AutomaticKeepAliveClientMixin {
```

Comment:
Why is this necessary?

Reply:
This is necessary to implement 'wantKeepAlive', which keeps the cards loaded even when they disappear from the screen due to scroll.